src/test/versionbits_tests.cpp

```diff
@@ -226,6 +267,13 @@ BOOST_AUTO_TEST_CASE(versionbits_test)
         // Make sure that no deployment tries to set an invalid bit.
         BOOST_CHECK_EQUAL(bitmask & ~(uint32_t)VERSIONBITS_TOP_MASK, bitmask);
 
+        // Check min_activation_height is on a retarget boundary
+        BOOST_CHECK_EQUAL(mainnetParams.vDeployments[i].min_activation_height % mainnetParams.nMinerConfirmationWindow, 0);
+        // Check min_activation_height is 0 for ALWAYS_ACTIVE and never active deployments
+        if (mainnetParams.vDeployments[i].nStartTime == Consensus::BIP9Deployment::ALWAYS_ACTIVE || mainnetParams.vDeployments[i].nStartTime == Consensus::BIP9Deployment::NEVER_ACTIVE) {
+            BOOST_CHECK_EQUAL(mainnetParams.vDeployments[i].min_activation_height, 0);
```

Comment:
```suggestion
            BOOST_CHECK_EQUAL(mainnetParams.vDeployments[i].min_activation_height, 0U);
```

Looks like you are hitting some stupid compiler error

Reply:
Fixed.